modelscope_agent/tools/wordart_tool.py

```diff
@@ -0,0 +1,156 @@
+import os
+import time
+
+import json
+import pandas as pd
+import requests
+from modelscope_agent.tools.tool import Tool, ToolSchema
+from pydantic import ValidationError
+from requests.exceptions import RequestException, Timeout
+
+MAX_RETRY_TIMES = 3
+
+
+class WordArtTexture(Tool):
+    description = '生成艺术字纹理图片'
+    name = 'wordart_texture_generation'
+    parameters: list = [{
+        'name': 'input.text.text_content',
+        'description': 'text that the user wants to convert to WordArt',
+        'required': True
+    }, {
+        'name': 'input.prompt',
+        'description':
+        'Users’ style requirements for word art may be requirements in terms of shape, color, entity, etc.',
+        'required': True
+    }]
+
+    def __init__(self, cfg={}):
+        self.cfg = cfg.get(self.name, {})
+        # remote call
+        self.url = 'https://dashscope.aliyuncs.com/api/v1/services/aigc/wordart/texture'
+        self.token = self.cfg.get('token', os.environ.get('API_TOKEN', ''))
```

Comment:
DASHSCOPE_API_TOKEN？

Reply:
wordart api token，written in tool_config or environ